lib/index.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
     'routes-segments-snake-case': require('./rules/routes-segments-snake-case'),
     'use-brace-expansion': require('./rules/use-brace-expansion'),
     'use-ember-get-and-set': require('./rules/use-ember-get-and-set'),
+    'avoid-leaking-callbacks-in-ember-objects': require('./rules/avoid-leaking-callbacks-in-ember-objects'),
```

Comment:
This should be alphabetical in the list.

Reply:
Move to the top in alphabetical order